reproducible/ubuntu/node_gen_tar.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+usage() {
+  echo "Usage: $0 [VERSION] [NODE_OUTPUT_TAR]"
+  echo
+  echo "[VERSION]: The semvar version of node to build ex: v8.5.0 ."
+  echo "[NODE_OUTPUT_PATH]: The path to output the compiled node rootfs."
+  echo
+  exit 1
+}
+
+if [ $# -ne 2 ]; then
+    usage
+fi
+
+VERSION=$1
+NODE_OUTPUT_PATH=$2
+
+# TODO(aaron-prindle) change this to use the exec method
+docker load -i $(pwd)/dockerfile_build/ubuntu_build.tar | awk '{print $3}'
```

Comment:
I don't think you'll be able to change this to exec - ubuntu_build.tar is an actual tar with no incremental loader.

Reply:
todo removed